src/Microsoft.AspNet.Mvc.ModelBinding/Validation/DataAnnotationsModelValidator.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
 {
-    public class DataAnnotationsModelValidator : IModelValidator, IClientModelValidator
+    public class DataAnnotationsModelValidator : IModelValidator
```

Comment:
@dougbu  
Huh? I do not understand our comment, this is not about adapters, this is about normal server side validation. the class that has been removed is `DataAnnotationsClientModelValidator`